hello_world/.gitignore

```diff
@@ -0,0 +1,244 @@
+
```

Comment:
why isn't this file in the root folder?

Reply:
Because the one in the root folder is generated by github when the repo is created and the other was generated when I ran `sam init`. I'll remove this one.